share/spack/gitlab/cloud_pipelines/stacks/developer-tools-manylinux2014/spack.yaml

```
@@ -93,7 +93,7 @@ spack:
   ci:
     pipeline-gen:
     - build-job:
-        image: ecpe4s/manylinux2014:2024.03.28
+        image: ghcr.io/spack/spack/manylinux2014:2024.03.28
```

This could be due to GitLab CI not parsing this slightly non-standard container uri correctly. In ghcr.io, the containers are under the organization, not the repository, so this would more conventionally be
```suggestion
        image: ghcr.io/spack/manylinux2014:2024.03.28
```
The way it is written now, the container name is `spack/manylinux2014`, see e.g. https://github.com/orgs/spack/packages/container/package/spack%2Fmanylinux2014. The slash in the name might confuse GitLab CI.

The access to https://github.com/orgs/spack/packages/container/package/spack%2Fmanylinux2014 seems fine (it's properly set to public, at least now it is, and I can pull it locally).

---

Thanks. I think the issue was actually that, when new packages are created for the first time, the visibility defaults to internal, thus requiring authorization to pull. I've changed the visibility to public. You can try:

```
$> docker pull ghcr.io/spack/spack/manylinux2014:2024.03.28
...
```

It should work. Indeed, the ref is `ghcr.io/spack/spack/manylinux2014:2024.03.28` with the `.../spack/spack/...`

I did not have permission to push images named `ghcr.io/spack/manylinux2014:2024.03.28`, I think because I am not an admin on the org, whereas I am an admin on the spack/spack repo. Do you know how I could push to ghcr.io/spack` instead of `ghcr.io/spack/spack`?